packages/core/src/modules/x509/X509ServiceOptions.ts

```diff
@@ -22,6 +22,7 @@ export interface X509CreateSelfSignedCertificateOptions {
   notBefore?: Date
   notAfter?: Date
   name?: string
```

Comment:
I think name can just be a string with `CN=X, N=X` I think that might be nicer to keep it like that. Otherwise we'll have to add a custom prop for everything

Reply:
or we need to nest / rename it